en/simulation/ros_interface.md

```diff
@@ -1,50 +1,53 @@
 # Interfacing the Simulation to ROS
 
-The simulated autopilot starts a second MAVLink interface on port 14557. Connecting MAVROS to this port allows to receive all data the vehicle would expose if in real flight.
+[ROS](../ros/README.md) (Robot Operating System) is a general purpose robotics library that can be used with the PX4 flight stack and also (optionally) with the Gazebo simulator. It uses the [MAVROS](../ros/mavros_installation.md) MAVLink node to communicate with PX4.
 
-## Launching MAVROS
+The diagram below shows the *standard* [PX4 simulation environment](../simulation/README.md#sitl-simulation-environment). PX4 communicates with the simulator (e.g. Gazebo) to receive sensor data from the simulated world and send motor and actuator values. It communicates with the GCS and Offboard API to send telemetry from the simulated environment and receive commands. 
 
-If an interface to ROS is wanted, the already running secondary MAVLink instance can be connected to ROS via [mavros](../ros/mavros_offboard.md). To connect to a specific IP (`fcu_url` is the IP / port of SITL), use a URL in this form:
+![PX4 SITL overview](../../assets/simulation/px4_sitl_overview.png)
 
-<div class="host-code"></div>
+The ROS/Gazebo integration with PX4 follows the pattern above. The only *slight* difference being that ROS initiates the connection on port 14557, while it is more typical for an offboard API to listen for connections on UDP port 14540. 
 
-```sh
-roslaunch mavros px4.launch fcu_url:="udp://:14540@192.168.1.36:14557"
-```
 
-To connect to localhost, use this URL:
+## Installing Gazebo for ROS
```

Comment:
Updated instructions to point to our build scripts, which include everything you need with PX4 in one easy script. Also link to the kinetic instructions for people who want to do in manually.

@TSC21 FYI, this will need an update once I know what happens with macOS standard installer. If the standard installer does not include ROS are there manual instructions that we know will work on macOS?

Reply:
Again, can't tell since I never tried to install ROS on a Mac.

Reply:
No worries - I'll ask on slack: https://px4.slack.com/archives/C0V533X4N/p1513730251000161

Reply:
FYI, so the simple answer is "we don't support it"